doc/databases.md

```diff
@@ -30,6 +30,20 @@ The databases do not need to be unpacked or prepared in any way after download.
 
 You can verify that they've been successfully downloaded (and view database properties such as `ksize` and `scaled`) with `sourmash sig summarize <output>`.
 
+## Sketches for human and animal genomes
+
+These include k=21, k=31, and k=51, at a scaled of 1000. Each file is about 110 MB.
```

Comment:
```suggestion
These signature files are representative sketches of model organisms. There suggested usage is to identify and remove contamination from environmental sketches. These include k=21, k=31, and k=51, at a scaled of 1000. Each file is about 110 MB.
```

Reply:
updated in dc9d4353 with:
+These sketches are of the latest releases of a number of animal
+genomes. Among other uses, they can be used to detect host
+contamination in microbial metagenomes.
+
+Each file includes sketches at k=21, k=31, and k=51, at a scaled of
+1000, and is about 110 MB.